### PR TITLE
Update hosted_happs in hpos-admin API to return stats

### PR DESCRIPTION
**Pull request by zo-el**
### `GET /hosted_happs`
####`200 OK`

```
{
    "hosted_happs": [
        {
            ...
            "stats": {
              "traffic": {
               "start_date": ISO8601, // start date of the first zome call tracked
               "total_zome_calls": u64, // total number of zome calls made
               "values": Vec<u64>  // graph data
              }
            }
       }
    ]
}
```

> Note: For the graph data, this data is built to display a graph for `zome_calls per day` since the start day

### CODE REVIEW

File: overlays/holo-nixpkgs/hpos-admin/hpos-admin.py [hunk 1]
```diff
@@ -89,14 +91,49 @@ def get_hosted_happs():
...
+        response = await websocket.recv()
+        print(response)
+        return  response.result
+
+def get_traffic_service_logger_call(instance_id):
+    return asyncio.get_event_loop().run_until_complete(hc_call('call', { "instance_id": instance_id ,"zome": "service", "function": "get_traffic", "args": {"filter": "DAY"} }))
+
+
+@app.route('/hosted_stats', methods=['GET'])
+def get_hosted_stats():
```
Comment (alastairong) [inline overlays/holo-nixpkgs/hpos-admin/hpos-admin.py:124]: I'm confused as to why we have basically duplicated the `/hosted_happs` endpoint here with `/hosted_stats`?

Shouldn't we just add 
`hosted_happ['stats'] = {"traffic": get_traffic_service_logger_call(hosted_happ['id']+"::servicelogger")}`
to `/hosted_happs` at line 102?

Comment (zo-el) [reply overlays/holo-nixpkgs/hpos-admin/hpos-admin.py]: Apologies this PR was not ready for review.. Now it is ready.

**Comment by zo-el** [comment]
Hi @peeech @alastairong, I do not have access to push this branch to develop. 
Can either one of you merge it?

**Comment by filalex77** [comment]
@zo-el If you rebase it onto latest develop, I can merge it.

**Comment by zo-el** [comment]
@filalex77, just rebased the branch

**Comment by filalex77** [comment]
@zo-el I need @alastairong to approve this PR first, as he has previously requested changes.

**Pull request merged**